x-pack/plugins/synthetics/public/apps/synthetics/components/step_details_page/hooks/use_network_timings_prev.ts

```diff
@@ -171,11 +180,16 @@ export const useNetworkTimingsPrevious24Hours = (stepIndexArg?: number) => {
     wait: aggs?.wait.values['50.0'] ?? 0,
     blocked: aggs?.blocked.values['50.0'] ?? 0,
     ssl: aggs?.ssl.values['50.0'] ?? 0,
+    transferSize: aggs?.transferSize.values['50.0'] ?? 0,
```

Comment:
So the baseline for the trends isn't the last test, but whether or not it's gone up from the median?

Just asking for clarity.

Reply:
Yes, media of last 24hours.